tools/rpm-bild.bash

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
Unportable since you cannot rely on bash being installed in `/bin`.
Since you don't use any bash features I would suggest changing this to `/bin/sh`.
Also the file extension should become `.sh`.

Reply:
Corrected.

Reply:
I meant a shebang for `#!/bin/sh`. Please add that back.